ci/licenses_golden/licenses_flutter

```diff
@@ -1481,6 +1481,7 @@ ORIGIN: ../../../flutter/impeller/renderer/gpu_tracer.cc + ../../../flutter/LICE
 ORIGIN: ../../../flutter/impeller/renderer/gpu_tracer.h + ../../../flutter/LICENSE
 ORIGIN: ../../../flutter/impeller/renderer/host_buffer.cc + ../../../flutter/LICENSE
 ORIGIN: ../../../flutter/impeller/renderer/host_buffer.h + ../../../flutter/LICENSE
+ORIGIN: ../../../flutter/impeller/renderer/mocks.h + ../../../flutter/LICENSE
```

Comment:
nit: maybe this belongs in the excluded one instead? I think we're doing that with at least some test-only files.

Reply:
Do you have a pointer to an example?  I can ask Hixie.

Reply:
moved it over, I think this makes sense